packages/slice-machine/src/hooks/usePromptToCreateContentExperiment.ts

```diff
@@ -6,5 +6,13 @@ export function usePromptToCreateContentExperiment(): UsePromptToCreateContentEx
   const variant = useExperimentVariant("slicemachine-prompt-to-create-content");
 
   // serve the experimental version to user with Telemetry deactivated
-  return { eligible: variant === undefined || variant?.value === "on" };
+  return {
```

Comment:
💡 #idea

I'm wondering, what about?

```suggestion
  return variant?.value !== "control"
```

Maybe less good to get when it's eligible?

Reply:
make sense, will change

Reply:
changed